addons/website/static/src/scss/website.edit_mode.scss

```diff
@@ -76,6 +76,8 @@ $-editor-messages-margin-x: 2%;
         &#wrap:empty, &#wrap > .oe_drop_zone.oe_insert:not(.oe_vertical):only-child {
             @extend %o-editor-messages;
             padding: 112px 0px;
+            pointer-events: none;
+            caret-color: transparent;
```

Comment:
This code (in `_startEditMode`): `this.$editorMessageElements.mousedown()` triggers a mousedown on the area to focus it. Can't this be done by removing this code only?
Be careful, in this case the editable area is still clickable (just unfocused)

Reply:
The snippets are supposed to bounce when clicking inside the empty area, your `pointer-events: none;` stuff prevents that. @sbuhl or do we remove the bounce effect added some weeks ago?

Also this does not seems to solve anything... the user enters edit mode, type text and it is still adding text inside the page... did you investigate what @xO-Tx said @oussamamessaoudi ?

Could you also review the commit message? I don't know what to expect at all with "remove cursor events in skeleton fragment"... something like "prevent users to type text  in empty pages" seems more readable?

Reply:
IMO: 
The proposed solution has a problem: if we still want the bounced effects.
About the solution proposed by @xO-Tx, the area will be unfocused and once clicked, the area will be editable.